packages/lesswrong/components/seasonal/solsticeSeason/SolsticeSeasonBanner.tsx

```diff
@@ -0,0 +1,503 @@
+import React, { useState, useMemo, useCallback, useEffect, useRef } from 'react';
+import { defineStyles, useStyles } from '../../hooks/useStyles';
+import { useQuery } from "@/lib/crud/useQuery";
+import { SuspenseWrapper } from '@/components/common/SuspenseWrapper';
+import { gql } from '@/lib/generated/gql-codegen';
+import { Link } from '@/lib/reactRouterWrapper';
+import classNames from 'classnames';
+import SolsticeGlobe3D from './SolsticeGlobe3D';
+import { SolsticeGlobePoint } from './types';
+import { FixedPositionEventPopup } from '../HomepageMap/HomepageCommunityMap';
+import { useIsAboveBreakpoint } from '@/components/hooks/useScreenWidth';
+
+const smallBreakpoint = 1525
+
+const styles = defineStyles("SolsticeSeasonBanner", (theme: ThemeType) => ({
+  root: {
+    position: 'fixed',
+    top: 0,
+    right: 0,
+    width: '50vw',
+    height: '100%',
+    [theme.breakpoints.down(1425)]: {
```

Comment:
what is with all the custom breakpoints?

Reply:
Frontpage banners generally require either custom breakpoints to look good.